server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/user/UserType.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.in.www1.hephaestus.codereview.user;
+
+public enum UserType {
```

Comment:
Technically I think the user type can also be organization. For now this is enough :)